packages/twenty-server/test/integration/metadata/suites/field-metadata/morph-relation/successful-add-one-target-to-metadata-morph-relation-v2.integration-spec.ts

```diff
@@ -0,0 +1,188 @@
+        icon: 'IconCompany',
+      },
+    });
+
+    createdObjectMetadataCompanyId = objectMetadataCompanyId;
+
+    const {
+      data: {
+        createOneObject: { id: objectMetadataOpportunityId },
+      },
+    } = await createOneObjectMetadata({
+      expectToFail: false,
+      input: {
+        nameSingular: 'opportunityForMorphRelationSecond',
+        namePlural: 'opportunitiesForMorphRelationSecond',
+        labelSingular: 'Opportunity For Morph Relation',
+        labelPlural: 'Opportunities For Morph Relation',
+        icon: 'IconOpportunity',
+      },
+    });
+
+    createdObjectMetadataOpportunityId = objectMetadataOpportunityId;
+
+    const {
+      data: {
+        createOneObject: { id: objectMetadataNewTargetId },
+      },
+    } = await createOneObjectMetadata({
+      expectToFail: false,
+      input: {
+        nameSingular: 'newTargetForMorphRelation',
+        namePlural: 'newTargetsForMorphRelation',
+        labelSingular: 'New Target For Morph Relation',
+        labelPlural: 'New Targets For Morph Relation',
+        icon: 'IconNewTarget',
+      },
+    });
+
+    createdObjectMetadataNewTargetId = objectMetadataNewTargetId;
+  });
+
+  afterAll(async () => {
+    const createdObjectMetadataIds = [
+      createdObjectMetadataPersonId,
+      createdObjectMetadataOpportunityId,
+      createdObjectMetadataCompanyId,
+      createdObjectMetadataNewTargetId,
+    ];
+
+    for (const objectMetadataId of createdObjectMetadataIds) {
+      await updateOneObjectMetadata({
+        expectToFail: false,
+        input: {
+          idToUpdate: objectMetadataId,
+          updatePayload: { isActive: false },
+        },
+      });
+      await deleteOneObjectMetadata({
+        expectToFail: false,
+        input: { idToDelete: objectMetadataId },
+      });
+    }
+  });
+
+  beforeEach(async () => {
+    const {
+      data: { createOneField: rawCreateOneField },
+    } = await createOneFieldMetadata({
+      input: {
+        label: 'field label',
+        name: 'fieldName',
+        objectMetadataId: createdObjectMetadataCompanyId,
+        description: 'Description for all',
+        type: FieldMetadataType.MORPH_RELATION,
+        morphRelationsCreationPayload: [
+          {
+            targetFieldIcon: 'Icon123',
+            targetFieldLabel: 'toto',
+            targetObjectMetadataId: createdObjectMetadataOpportunityId,
+            type: RelationType.MANY_TO_ONE,
+          },
+          {
+            targetFieldIcon: 'Icon123',
+            targetFieldLabel: 'tata',
+            targetObjectMetadataId: createdObjectMetadataPersonId,
+            type: RelationType.MANY_TO_ONE,
+          },
+        ],
+      },
+      expectToFail: false,
+    });
+
+    createdFieldMetadataId = rawCreateOneField.id;
+  });
+
+  it('When adding one object metadata target to pre-existing morph relation, the relation on the target object metadata should be updated', async () => {
```

Comment:
is there any failing case? like putting the same object on both side of the relation, or putting an object twice?